base/collect_database.go

```diff
@@ -99,7 +101,7 @@ func getElapsedTime(versionInfo map[string]interface{}, exp *Experiment) (int64,
 		}
 	}
 
-	// calculate the ElapsedTime based on the StartingTime if it has been provided
+	// calculate the ElapsedTime based on the startingTime if it has been provided
```

Comment:
What is the units for `elapsedTime`? Is it seconds? Nanoseconds? Something else?